### PR TITLE
Add Total Count for IndexView, Search Page

### PR DESCRIPTION
**Pull request by tboyetteredhat**
# [OSIDB-2438] [Add count of results returned]

## Checklist:
- [x] Linting passed
- [x] Type checks passed
- [x] Tests suite passed
- [x] Commits consolidated
- [x] Changelog updated
- [x] Test cases added/updated
- [x] Jira ticket updated

## Summary:

Add Total, Loaded Count for IssueQueue

## Changes:

Add Total Count for IssueQueue table

### CODE REVIEW

**Review by C-Valen — APPROVED** [review]
LGTM

I'd recommend adding bold font type to make it clearer to the user.

And I'd place it within "My Issues" and "Open Issues" check-boxes, but that's just a personal preference.

image

File: src/components/IssueQueue.vue [hunk 1]
```diff
@@ -157,6 +157,7 @@ watch(params, () => {
         <span class="visually-hidden">Loading...</span>
       </span>
       <span v-if="isLoading"> Loading Flaws&hellip; </span>
+      <span class="float-end fw-bold"> Total: {{ issues.length }}</span>
```
Comment (superbuggy) [inline src/components/IssueQueue.vue:160]: I think we should show the number of fetched results (`issues.length`) and the `.count` from the response from GET requests to `https://osidb-stage.prodsec.redhat.com/osidb/api/v1/flaws`

Comment (superbuggy) [reply src/components/IssueQueue.vue:160]: https://github.com/RedHatProductSecurity/osim/blob/fix/OSIDB-2390-sequential-saving-on-flaws/src/composables/useFlaws.ts#L17-L63

You can access `response.data.count` in the highlighted block above and use a `ref` to capture the number of results found by OSIDB.

**Review by superbuggy — CHANGES_REQUESTED** [review]

File: src/components/IssueQueue.vue (hunk 1 above)
Comment (tboyetteredhat) [reply src/components/IssueQueue.vue:160]: Thanks for your feedback. I will update to show both total and loaded count.

**Review by d10n — APPROVED** [review]

**Review by superbuggy — APPROVED** [review]

**Pull request merged**